From 9fdb318822206e143e8be453b986c470823dc16d Mon Sep 17 00:00:00 2001
From: Chris McDonald <cjmcdonald@chromium.org>
Date: Fri, 22 Jan 2021 14:27:57 -0800
Subject: [PATCH] CHROMIUM: Fix cross-compilation

Updating cmake-utils.eclass causes this build to break due to an intermediate
binary not being visible on the PATH during compilation. This patch specifies
that the binary should be used from the CWD during the build, which resolves
the issue. This patch needs to be included and carried forward until the
upstream bug is resolved.

Upstream-Bug: https://github.com/KhronosGroup/Vulkan-Loader/issues/249
---
 loader/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/loader/CMakeLists.txt b/loader/CMakeLists.txt
index 7cc5fd71e..70288cd3d 100644
--- a/loader/CMakeLists.txt
+++ b/loader/CMakeLists.txt
@@ -191,7 +191,7 @@ else(UNIX AND NOT APPLE) # i.e.: Linux
         set(OPT_LOADER_SRCS ${OPT_LOADER_SRCS} unknown_ext_chain_gas.S)
         add_executable(asm_offset asm_offset.c)
         target_link_libraries(asm_offset Vulkan::Headers)
-        add_custom_command(OUTPUT gen_defines.asm DEPENDS asm_offset COMMAND asm_offset GAS)
+        add_custom_command(OUTPUT gen_defines.asm DEPENDS asm_offset COMMAND ./asm_offset GAS)
         add_custom_target(loader_asm_gen_files DEPENDS gen_defines.asm)
         target_compile_definitions(asm_offset PRIVATE _XOPEN_SOURCE=500) # hush compiler warnings for readlink
     else()
-- 
2.30.0.280.ga3ce27912f-goog

